index.css

```diff
@@ -0,0 +1,309 @@
+body * {
+  box-sizing: border-box;
+  font-family: 'Roboto';
+}
+
+nav {
+  display: grid;
+  grid-template-columns: repeat(7, 1fr);
+  grid-template-rows: 150px;
+  background: rgb(211, 58, 44);
+}
+
+.headerBox {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: white;
+  }
+
+.headerBox h3 {
+  width: min-content;
+}
+
+#firstSection {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: 790px 790px;
+  background: gray;
+}
+
+.firstSectionFloat{
+  margin-top: 50px;
+  position: absolute;
+  text-align: center;
+  width: 100%;
+}
+.fs1{
+  background: #b13125;
+}
+
+.fs2 {
+  background: rgb(188, 52, 40);
+}
+
+.fs3{
+  background: #b13125;
+}
+
+.fs4 {
+  background: rgb(188, 52, 40);
+}
+
+#titleFloat{
+  display: inline-block;
+  color: white;
+  font-size: 26px;
+  font-weight: bold;
+  background: black;
+  padding: 5px;
+}
+
+.firstSectionBox {
+  display: flex;
+  border: solid black 1px;
+  align-items: center;
+  justify-content: center;
+}
+
+.firstSectionBox h1 {
+  width: 80%;
+  text-align: center;
+  font-size: 30px;
+  letter-spacing: 2px;
+  color: white;
+}
+
+#secondSection {
+  border: solid black 1px;
+  color: black;
+  background: #f6f3f2;
+  font-size: 20px;
+}
+
+.ss21, .ss33 {
```

Comment:
Some rafactoring: 
Since ss21, ss33 have the same properties, we need just one of them with more descriptive name like `two-third-box`. That way we can reuse it in any components that need the same styling.